### ISSUE TITLE
Yale lock

### ISSUE THREAD

**Issue by hypascend** [damianxd/homebridge-vera#106]
Hi,

Every so often my phone buzzes with door unlocked and then locked again, yet no one has unlocked the door,  Sometimes it does this 4-5 times in succession and then not at all.  It also seems to do this when you wake up some other device, like a thermostat or lights, etc.

**Comment by damianxd** [comment]
I have a yale lock too and sometimes I have a similar problem. Mostly because Vera sometimes sends a lock open value while HAP check for changes.

I did stop that behavior on the latest versions of this bridge. But if you continue with the problem, it is more likely the problem I've explained and currently, I'm trying to find a way to get realtime data from Vera without depending on sequential request.
